src/pages/lib/auth/emailpassword/q/platform/[platform].mdx

```diff
@@ -1,12 +1,333 @@
+
+export async function signIn({ username, password }: SignInParameters) {
+  try {
+    const user = await Auth.signIn(username, password);
+  } catch (error) {
+    console.log('error signing in', error);
+  }
+}
+```
+</Block>
+<Block name="JavaScript">
+
+```js
+import { Auth } from 'aws-amplify';
+
+async function signIn() {
+  try {
+    const user = await Auth.signIn(username, password);
+  } catch (error) {
+    console.log('error signing in', error);
+  }
+}
+```
+</Block>
+</BlockSwitcher>
+
+### Sign out
+
+To set up local sign-out you use the `signOut` method of the Auth class.
+
+<BlockSwitcher>
+<Block name="TypeScript">
+
+```ts
+import { Auth } from 'aws-amplify';
+
+export async function signOut() {
+  try {
+    await Auth.signOut();
+  } catch (error) {
+    console.log('error signing out: ', error);
+  }
+}
+```
+</Block>
+<Block name="JavaScript">
+
+```js
+import { Auth } from 'aws-amplify';
+
+async function signOut() {
+  try {
+    await Auth.signOut();
+  } catch (error) {
+    console.log('error signing out: ', error);
+  }
+}
+```
+</Block>
+</BlockSwitcher>
+
+You can sign out users from all devices by adding global sign-out. It also invalidates all refresh tokens issued to an user. The user's current access and Id tokens remain valid on other devices until the refresh token expires (access and Id tokens expire one hour after they are issued).
+
+<BlockSwitcher>
+<Block name="TypeScript">
+
+```ts
+import { Auth } from 'aws-amplify';
+
+export async function signOut() {
+  try {
+    await Auth.signOut({ global: true });
+  } catch (error) {
+    console.log('error signing out: ', error);
+  }
+}
+```
+</Block>
+<Block name="JavaScript">
+
+```js
+import { Auth } from 'aws-amplify';
+
+async function signOut() {
+  try {
+    await Auth.signOut({ global: true });
+  } catch (error) {
+    console.log('error signing out: ', error);
+  }
+}
+```
+</Block>
+</BlockSwitcher>
+
+Once you implement the Amplify Library methods within your application, you can test the sign-up, sign-in, and sign-out functionality.
+
+### View the created user in the AWS Console
+
+After you test the sign-up, sign-in, and sign-out functionality, you can take a look at the created user in the Amazon Cognito console. To do so, you can use `amplify console auth` to open the console directly from the CLI, or take a look at the existing User Pools in the console. Note the users status should be `CONFIRMED` if you successfully verified the user; however, it can be in `unconfirmed` if the user is not yet verified. You can also check to make sure any other required attributes are updating correctly.
+
+You now have enabled sign-up, sign-in, and sign-out for your authentication and verified its functionality with a test user.
+
+## Conclusion
+
+Congratulations! You finished the **Enable sign-up, sign-in, sign-out** guide. In this guide, you enabled sign-up, sign-in, and sign-out with the Amplify Libraries. You also verified they function as expected.
```

Comment:
Seems slightly redundant. Maybe: 

Congratulations! You finished the **Enable sign-up, sign-in, sign-out** guide. You are now able to utilize the associated methods within the Authentication category and have verified they function as expected.

Reply:
```suggestion
Congratulations! You finished the **Enable sign-up, sign-in, sign-out** guide and verified the functionality works as expected.
```